eng/build/Workers.Powershell.props

```
@@ -8,8 +8,13 @@
 
   <Target Name="RemovePowershellWorkerRuntimes" BeforeTargets="AssignTargetPaths" Condition="'$(RuntimeIdentifier)' != ''">
     <ItemGroup>
-      <_KeepPowerShellRuntime Include="win;win-x86;win10-x86;win-x64;win10-x64" Condition="$(RuntimeIdentifier.StartsWith(win))" />
-      <_KeepPowerShellRuntime Include="linux;linux-x64;unix" Condition="$(RuntimeIdentifier.StartsWith(linux))" />
+      <_KeepPowerShellRuntime Include="win;win-x86;win10-x86;win-x64;win10-x64" Condition="$(RuntimeIdentifier.StartsWith('win')) and !$(RuntimeIdentifier.Contains('arm'))" />
+      <_KeepPowerShellRuntime Include="linux;linux-x64;unix" Condition="$(RuntimeIdentifier.StartsWith('linux')) and !$(RuntimeIdentifier.Contains('arm'))" />
```

Maybe we can be explicit for all runtime IDs except for windows.
```suggestion
      <_KeepPowerShellRuntime Include="linux;linux-x64;unix" Condition="'$(RuntimeIdentifier)' == 'linux-x64'" />
```

---

By this you mean:

```xml
<_KeepPowerShellRuntime Include="linux" Condition="'$(RuntimeIdentifier)' == 'linux'" />
<_KeepPowerShellRuntime Include="linux-x64" Condition="'$(RuntimeIdentifier)' == 'linux-x64'" />
<_KeepPowerShellRuntime Include="unix" Condition="'$(RuntimeIdentifier)' == 'unix'" />
```

or would unix and linux be:

<_KeepPowerShellRuntime Include="linux;unix" Condition="'$(RuntimeIdentifier)' == 'linux'" />

leaving x64 and arm to be separate

---

The idea would be to get a point of given a `RuntimeIdentifier`, populate `_KeepPowerShellRuntime` with the list of runtime folders needed for that runtime. The answer to that varies based on the runtime. To run `linux-x64`, I think all of `unix;linux;linux-x64` are needed. I don't know what linux-arm64 and osx need